tests/result/imaps.pcap.out

```diff
@@ -1,29 +1,32 @@
-Guessed flow protos:	0
+Guessed flow protos:	1
```

Comment:
Shouldn't there be a `0` ?

Reply:
The flow is too short, now, to have a "not guessed" classification, because the packets with the certificate are not present.
For IMAPS/POPS/SMTPS flows before this change we only needed the Client Hello (basically), now we need the full TLS handshake, like the "standard" TLS flows. But we don't have it in these 3 traces.

Bottom line: the flows ends (i.e. we don't have more packets) while we are still classifying it.